src/neo/SmartContract/Native/Tokens/Nep11AccountState.cs

```diff
@@ -0,0 +1,21 @@
+using Neo.VM;
+using Neo.VM.Types;
+using System.Numerics;
+
+namespace Neo.SmartContract.Native.Tokens
+{
+    public class Nep11AccountState : IInteroperable
```

Comment:
It's equal to `Nep5AccountState` we should rename it and share it

Reply:
Agree. We reviewed almost at the same time. aehauhea ajaj